tests/Unit/Domain/ShoppingListTest.php

```diff
@@ -15,12 +15,13 @@ public function testConstructWithoutList(): ShoppingList
     {
         $list = new ShoppingList(
             $slug = new Slug('my-list'),
-            'My List',
+            'My List', null,false
         );
 
         $this->assertSame($slug, $list->getSlug());
         $this->assertSame('My List', $list->getName());
         $this->assertEmpty($list->getItems()->all());
+        $this->assertFalse($list->isArchived());
```

Comment:
Comment: we also need to test somewhere that `isArchived()` returns `true` if the list is constructed with `true` for that property.